clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/wgscram/TransferWgsCramV1Metadata.scala

```diff
@@ -69,15 +70,20 @@ case class TransferWgsCramV1Metadata(
 
   // As of DSDEGP-1711, we are only delivering the cram, crai, and md5
   override def mapMove(
+    samplePrefix: Option[String] = None,
     pathMapper: Option[URI] => Option[URI]
   ): TransferWgsCramV1Metadata = {
+    val prefixedCram = cramPath.map { cp =>
+      val name = new File(cp.getPath).getName
+      val directory = URI.create(cp.toString.dropRight(name.length))
+      directory.resolve(s"${samplePrefix.getOrElse("")}$name")
```

Comment:
Isn't `pathMapper` already effectively doing `directory.resolve(<file name>)`?

Reply:
Yes, via `moveIntoDirectory()`, but that assumes `pathsToDelete` are also the ones to prefix.

Reply:
Hm? `moveIntoDirectory` isn't tied to `pathsToDelete`, it's just doing a `destination.resolve(<filename>)` (so it's duplicate work to also do the resolve here).